### ISSUE TITLE
support Batch Norm (beta gamma)?

### ISSUE THREAD

**Issue by gd-zhang** [tensorflow/kfac#11]
Currently, Batch Norm is not supported. It would be great if `layer_collection.register_bn` is added.

**Comment by jinxin0924** [comment]
Hi, @james-martens , any plan for BN?

**Comment by james-martens** [comment]
Hi,

You can use KFAC with non-standard types of layer including Batch Norm
layers by registering the parameters as "generic".

Also note that you should be using https://github.com/tensorflow/kfac
instead of the (deprecated) version released with TensorFlow.  Maybe you
already are.

James
